sceptre/template.py

```diff
@@ -175,9 +176,14 @@ def upload_to_s3(self):
             }
         )
 
-        url = "https://{0}.s3.amazonaws.com/{1}".format(
-            bucket_name, bucket_key
-        )
+        if bucket_region in ["cn-north-1", "cn-northwest-1"]:
```

Comment:
could you move the list of regions for china region e.g.

```python
china_regions = ["cn-north-1", "cn-northwest-1"]

if bucket_region in china_regions:
...
```

Reply:
added